include/iarc7_fc_comms/CommonConf.hpp

```diff
@@ -27,7 +27,7 @@ enum class FcCommsStatus
 
 struct CommonConf
 {
-    static constexpr const float kFcSensorsUpdateRateHz{400};
+    static constexpr const float kFcSensorsUpdateRateHz{100};
```

Comment:
What was the reasoning behind the drop in update rate? And in a future revision can this be made to be param loaded?

Reply:
Oversampling helps reduce Gaussian noise but is placing an extremely high load on the kalmann filter. But, all controls are moving to be less dependent on the accelerometer so this is no longer necessary.